lib/modules/manager/kustomize/extract.ts

```diff
@@ -143,13 +144,26 @@ export function extractHelmChart(
   if (!helmChart.name) {
     return null;
   }
-
-  return {
+  const res: PackageDependency = {
     depName: helmChart.name,
     currentValue: helmChart.version,
     registryUrls: [helmChart.repo],
     datasource: HelmDatasource.id,
+    depType: 'HelmChart',
   };
+
+  if (isOCIRegistry(helmChart.repo)) {
+    res.datasource = DockerDatasource.id;
+    const repoUrl = helmChart.repo.replace('oci://', '');
+    res.packageName = `${repoUrl}/${helmChart.name}`;
+    res.registryUrls = [`https://${repoUrl}`];
```

Comment:
```suggestion
```

not needed, is areleady encoded inside `packageName`

Reply:
Right, but the value of `res.registryUrls` is still set on line 150, so without this, the packageName and registryUrls are inconsistent. It'll be `oci://` for one but not the other. I think it's best to use `https://` in all cases.

Reply:
Simply check for OCIRegistry first and return inside the if block

Reply:
I don't see how that changes anything.

The question I'm asking is simple: What should `registryUrls` look like for an OCI chart?

Reply:
Actually, I see what you meant, I can just return separately in each case. 👍